src/Data/Gdpr/GdprEditingConfig.ts

```diff
@@ -1,9 +1,4 @@
 import { provideEditingConfig } from 'scrivito'
 import { Gdpr } from './GdprDataClass'
 
-provideEditingConfig(Gdpr, {
-  title: 'GDPR consent',
-  attributes: {
-    active: { title: 'Active?' },
-  },
-})
+provideEditingConfig(Gdpr, { title: 'GDPR consent' })
```

Comment:
Not related to this file: The `GeneralMarketingConsent` still has a missing "boolean" attribute for `isConsentGiven`.

Same for `Subscription`.